docs/reference/analysis/tokenfilters/synonym-graph-tokenfilter.asciidoc

```diff
@@ -69,10 +69,10 @@ PUT /test_index
                     }
                 },
                 "filter" : {
-                	"my_stop": {
-                		"type" : "stop",
-                		"stopwords": ["bar"]
-                	},
```

Comment:
These were tabs, see https://www.elastic.co/guide/en/elasticsearch/reference/master/analysis-synonym-tokenfilter.html to check how this looks like now. Replacing with whitespaces so indention is corrected.